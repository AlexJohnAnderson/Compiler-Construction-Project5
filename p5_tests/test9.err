FATAL [2,5]-[2,15]: Invalid equality operation
Segmentation fault (core dumped)
