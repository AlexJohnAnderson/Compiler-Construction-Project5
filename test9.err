Only 1 input file allowedtest9.dg
Usage: dgc <infile>
 [-t <tokensFile>]: Output tokens to <tokensFile>
 [-p]: Parse the input to check syntax
 [-u <unparseFile>]: Output canonical program form
 [-n <nameFile>]: Output program with IDs annotated with symbols
 [-c]: Perform type analysis / typecheck the program
