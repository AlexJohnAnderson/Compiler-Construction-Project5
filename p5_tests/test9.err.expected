FATAL [2,5]-[2,15]: Invalid equality operation
Type Analysis Failed
